index.php

```diff
@@ -0,0 +1,364 @@
+<!DOCTYPE HTML>
+<!-- Author: Ashley Burnett -->
+
+<?php
+//Used for browser title and for META tags.
+$title = "Medieval Latin Lessons - The National Archives"
+?>
+
+<!-- no-js class is needed for Modernizr -->
+<html lang="en-gb" class="no-js">
+<head>
+    <title><?= $title ?></title>
+	<?php
+	include 'inc/head/head.php';
+	?>
+</head>
+<body>
+<?php
+include 'inc/body/body.php';
+include 'functions/generate-breadcrumbs.php';
+
+//		Add breadcrumbs here
+$pagesArray = array(
+	array( "Help with your research", "http://nationalarchives.gov.uk/help-with-your-research/" ),
+	array( "Medieval Latin Lessons" )
+);
+
+generate_breadcrumbs( $pagesArray );
+?>
+
+<!-- BEGIN EDITING CONTENT -->
+<div class="level-one">
+    <div class="container">
+        <div class="row" role="banner">
+            <div class="col-md-12">
+                <article class="banner feature-img feature-img-bg"
+                         style="background-image: url(./img/banner-image.png);">
+                    <div class="entry-header">
+                        <h1>Latin 1086 – 1733</h1>
+                    </div>
+                </article>
+            </div>
+        </div>
+
+    </div>
+</div>
+
+<main id="primary" role="main" class="content-area">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <article>
+                    <div class="entry-header">
+                        <h1>Free Medieval Latin Lessons</h1>
+                    </div>
+                    <div class="row entry-content">
+                        <div class="col-md-12">
+                            <p>This tutorial is a beginners' guide to the Latin used in documents between 1086 and
+                                1733.</p>
+                        </div>
+                    </div>
+                </article>
+
+            </div>
+        </div>
+        <div class="row equal-heights" id="equal-heights">
+            <div class="col-xs-12 col-sm-6">
+                <article>
+                    <div class="entry-header">
+                        <h2>
+                            <a href="#" title="#">Latin Stage 1</a>
+                        </h2>
+                    </div>
+                    <div class="entry-content clearfix">
+                        <ul class="sibling">
+                            <li>Lesson 1
+                                <ul class="child">
+                                    <a href="#">
```

Comment:
As discussed, please add the `<a>` inside the `<li>` items.
So it should look like this `<li><a href="#">Lesson Number</a></li>`.